scripts/generate-assets.ts

```diff
@@ -53,7 +55,11 @@ async function processDirectory(assets: Record<string, string>, dir: string) {
                 baseUri: target,
                 parseResolveResult(opts) {
                   return new Promise<IUriParserResult>(resolve => {
-                    resolve({ result: parse(opts.result) });
+                    try {
+                      resolve({ result: parse(opts.result) });
+                    } catch (e) {
+                      resolve({ error: e });
```

Comment:
Can we reject?

Reply:
@P0lip Fixed.

I got lured by the `IUriParserResult` type definition. But, indeed, it looks like the `error` property is completely ignored by the code.

https://github.com/stoplightio/json-ref-resolver/blob/beb48be41af65a94c870989aecf2b6ea00607b8e/src/runner.ts#L394-L403